api/filters.py

```diff
@@ -12,99 +12,115 @@
 
 logger = logging.getLogger(__name__)
 
-NULL_VALUE = 'unspecified'
+NULL_VALUE = "unspecified"
```

Comment:
From the demo we had earlier, I believe this has been properly implemented. 
However, the changes you've included in relation to formatting are also in this PR -> `https://github.com/AndelaOSP/art-backend/pull/292`. I'd recommend you exclude them from this PR. Merge conflicts are also likely to occur once the other one is merged.

Reply:
@RonKbS this is a file change by black

Reply:
Yes it is. However, there was a PR that's been merged. It was meant to deal with the formatting issues. => ``https://github.com/AndelaOSP/art-backend/pull/292`. You'll probably need to look into undoing the current changes as well. You have 65 files worked on now.